www/~/%username/subscriptions.json.spt

```diff
@@ -0,0 +1,41 @@
+""" Get or change authenticated user's subscriptions.
+"""
+from aspen import Response
+from gratipay.exceptions import NoTeam
+
+[------------------------------------------------------------------------]
+
+if user.ANON:
+    raise Response(403, _("Please sign in first."))
+
+else:
+    participant = user.participant
```

Comment:
Use [`get_participant`](https://github.com/gratipay/gratipay.com/blob/master/gratipay/utils/__init__.py#L56) here to make sure the user exists, and that the authenticated user has the necessary permissions to access the endpoint. The current way this is implemented, looks like hitting `www/~/aandis/subscriptions.json.spt` is not going to give me (logged in as `rohitpaulk`) a 403/404, it is going to return my own subscriptions.